openslides/core/static/js/core/start.js

```diff
@@ -4,31 +4,55 @@
 
 angular.module('OpenSlidesApp.core.start', [])
 
-.run([
+.factory('OS', [
```

Comment:
I would prefer just `OpenSlides` because this is the main factory where to start and to end.